lang/py3/avro/schema.py

```diff
@@ -385,7 +385,7 @@ def NewWithDefaultNamespace(self, namespace):
     """
     return Names(names=self._names, default_namespace=namespace)
 
-  def GetName(self, name, namespace=None):
+  def _get_name(self, name, namespace=None):
```

Comment:
I am a bit conflicted on this. I see that this is the only usage, but I am unsure how people use it. 
@rdblue  can you check this out and let us know if moving a function as an internal use only in a minor is okay.
Else, +1